cashctrl_ledger/tests/test_accounts.py

```diff
@@ -14,165 +14,139 @@
     /Balance,         9990,      EUR,         , Test EUR Bank Account
     /Balance/Node,    9993,      EUR,         , Transitory Account EUR
 """
-
-LEDGER_CSV = """
-    id,     date, account, contra, currency,     amount, report_amount, description
-    1,  2024-01-21,  9992,   9995,      CHF,     100.00,              , transaction 1
-    2,  2024-02-22,  9991,   9994,      USD,     100.00,         88.88, transaction 2
-    3,  2024-03-23,  9991,       ,      USD,     100.00,         85.55, transaction 3
-    3,  2024-03-23,      ,   9995,      CHF,      85.55,              , transaction 3
-    4,  2024-04-24,  9994,   9991,      USD,     100.00,         77.77, transaction 4
-    5,  2024-05-25,  9992,   9995,      CHF,      10.00,              , transaction 5
-    6,  2024-06-26,  9995,       ,      CHF,      95.55,              , transaction 6
-    6,  2024-06-26,      ,   9991,      USD,     100.00,         95.55, transaction 6
-"""
-LEDGER_ENTRIES = pd.read_csv(StringIO(LEDGER_CSV), skipinitialspace=True)
 ACCOUNTS = pd.read_csv(StringIO(ACCOUNT_CSV), skipinitialspace=True)
 
 
 class TestAccounts(BaseTestAccounts):
+    ACCOUNTS = BaseTestAccounts.ACCOUNTS.copy()
+    # Set the default root node for CashCtrl. In CashCtrl it is not possible to create root nodes
+    ACCOUNTS.loc[:, "group"] = "/Assets"
+    # TODO: Remove when Assets will be implemented
+    ACCOUNTS.loc[ACCOUNTS["currency"] == "JPY", "currency"] = "USD"
+    # TODO: Remove when error with .ne() will be fixed
+    ACCOUNTS.loc[:, "tax_code"] = pd.NA
+
+    TAX_CODES = BaseTestAccounts.TAX_CODES.copy()
+    # In CashCtrl it is not possible to create TAX CODE without specified account
+    TAX_CODES = TAX_CODES[~(TAX_CODES["id"] == "EXEMPT")]
```

Comment:
Yep. I've encountered the same problem in real life.

My preferred solution (also in real life) is to set an account, which satisfies CashCtrl but is meaningless (the rate is zero and hence all tax amounts that are supposed to be booked onto this account are zero). Please use the same account that we use for input tax.

```
TAX_CODES.loc[TAX_CODES["account"].isna(), "account"] = TAX_CODES.query["id == 'IN_STD']["account"]
```

Instead of:
```
TAX_CODES = TAX_CODES[~(TAX_CODES["id"] == "EXEMPT")]
```

(Btw: try to avoid making implicit assumptions when coding, such as that there is a tax code named 'EXEMPT' that has no account assigned. Go for the explicit version instead and drop or modify all TAX_CODES where the account is NA.)